dir.props

```diff
@@ -274,6 +274,10 @@
 
   <!-- Setup properties per OSGroup -->
   <Choose>
+    <When Condition="'$(IsReferenceAssembly)' == 'true' OR'$(MSBuildProjectExtension)' == '.pkgproj'">
```

Comment:
They are, but when resolved through a projectreference (eg: partial facade) they get the OSGroup of the facade project, which is sometimes OS-specific.